python/hsfs/core/arrow_flight_client.py

```diff
@@ -566,8 +573,25 @@ def _serialize_featuregroup_connector(fg, query, on_demand_fg_aliases):
                     connector["filters"] = _serialize_filter_expression(
                         join_obj._query._filter, join_obj._query, True
                     )
+    elif fg.time_travel_format == "DELTA":
+        connector["time_travel_type"] = "delta"
+        connector["type"] = fg.storage_connector.type
+        connector["options"] = fg.storage_connector.connector_options()
+        if fg.storage_connector.type == StorageConnector.S3:
+            connector["path"] = fg.location
```

Comment:
shouldn't `path` be under `options`?

Reply:
right!

Reply:
Actually, i think `path` should be on the same level as `query`. Since i believe it is either one or the other. Therefore, maybe adapting flying duck would be more consistent.

Reply:
It should be in connector level because you may join delta fg